README.md

```diff
@@ -2,7 +2,8 @@
 
 1. Create a fork of this project
 2. Clone your fork into your local machine.
-3. Add an `upstream` remote pointing to the main CYF repo (to be able to pull updates from your team members)
+3. Add an `upstream` remote pointing to the main CYF repo (to be able to pull updates from your team members).
+   Git remote add upstream git@github.com:CodeYourFuture/scot-broca-translators.git.
```

Comment:
format in a code block, and make sure it is `git` instead of `Git`

Reply:
I changed Git to git